assets/l10n/app_en.arb

```diff
@@ -27,18 +31,298 @@
+    "placeholders": {
+      "filename": {
+        "type": "String",
+        "example": "file.txt"
+      }
+    }
+  },
+  "contentValidationErrorTooLong": "Message length shouldn't be greater than 10000 characters.",
+  "@contentValidationErrorTooLong": {
+    "description": "Content validation error message when the message is too long."
+  },
+  "contentValidationErrorEmpty": "You have nothing to send!",
+  "@contentValidationErrorEmpty": {
+    "description": "Content validation error message when the message is empty."
+  },
+  "contentValidationErrorQuoteAndReplyInProgress": "Please wait for the quotation to complete.",
+  "@contentValidationErrorQuoteAndReplyInProgress": {
+    "description": "Content validation error message when a quotation has not completed yet."
+  },
+  "contentValidationErrorUploadInProgress": "Please wait for the upload to complete.",
+  "@contentValidationErrorUploadInProgress": {
+    "description": "Content validation error message when attachments have not finished uploading."
+  },
+  "suggestedActionDialogCancel": "Cancel",
+  "@suggestedActionDialogCancel": {
+    "description": "Button label to cancel a suggested action."
+  },
+  "suggestedActionDialogContinue": "Continue",
+  "@suggestedActionDialogContinue": {
+    "description": "Button label in accept a suggested action."
+  },
+  "errorDialogContinue": "OK",
+  "@errorDialogContinue": {
+    "description": "Button label in error dialogs to acknowledge error."
+  },
+  "errorDialogTitle": "Error",
+  "@errorDialogTitle": {
+    "description": "Generic title for error dialog."
+  },
+  "lightboxCopyLinkTooltip": "Copy link",
+  "@lightboxCopyLinkTooltip": {
+    "description": "Tooltip in lightbox for the copy link action."
+  },
+  "loginPageTitle": "Log in",
+  "@loginPageTitle": {
+    "description": "Page title for login page."
+  },
+  "loginFormSubmitLabel": "Log in",
+  "@loginFormSubmitLabel": {
+    "description": "Button text to submit login credentials."
+  },
+  "loginAddAnAccount": "Add an account",
+  "@loginAddAnAccount": {
+    "description": "Page title for screen to add a zulip account."
+  },
+  "loginAddAnAccountConfirm": "Continue",
+  "@loginAddAnAccountConfirm": {
+    "description": "Button label to submit form for adding an account."
+  },
+  "loginServerUrlInputLabel": "Your Zulip server URL",
+  "@loginServerUrlInputLabel": {
+    "description": "Input label in login page for Zulip server URL entry."
+  },
+  "loginHidePassword": "Hide password",
+  "@loginHidePassword": {
+    "description": "Icon label for button to hide password in input form."
+  },
+  "loginValidationPassword": "Please enter your password",
+  "@loginValidationPassword": {
+    "description": "Prompt for input for password field."
+  },
+  "loginValidationPasswordLabel": "Password",
+  "@loginValidationPasswordLabel": {
+    "description": "Label for input for password field."
```

Comment:
Is this about validation? I think it isn't — it's just the label on the "password" field at login.

Similarly for email and username below.